css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-color: #0090e0;
+  font-family: Lato, Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+  background-color: #0090e0;
+  color: white;
+}
+
+.column-third {
+  height: 110px;
+  width: 110px;
```

Comment:
The `width` or `flex-basis` needs to match the name given to the class. In this case, it should have a value of `33%`

Reply:
Also, as I mentioned in my last review, the **_only_** declaration on a `column` class rule should be either `width` or `flex-basis`. All the rest of this styling should be applied to a semantic class, or made into additional utility classes.